### PR TITLE
Added python engineer to open positions

### LINKED ISSUE #7: We are hiring Backend Engineer [Python]
Please make necessary changes @ChillarAnand

### PR DESCRIPTION
**Pull request by ChillarAnand**
closes #7

### CODE REVIEW

**Comment by subho007** [comment]
LGTM 👍

**Comment by yashwin** [comment]
LGTM 👍

**Comment by hemansutanty** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -80,7 +81,7 @@ The interview process in the order:
 
 1. 3 programming challenges (If your GitHub/Bitbucket/StackOverflow profile is impressive or if have an impressive portfolio - we will not ask you to take these challenges. You can skip this, as you have already proven yourself, and get to next step)
 1. Phone/Skype/Hangouts interview (personal + past experiences + little bit of tech) with our Tech Team
-1. A half-day pair programming with the entire tech team (If you are eligile to skip the first step - you can skip this step as well, if you want)
+1. Small project. We will give you a small project which can be done in a week. You can send it once you have completed the project. You can reach us anytime if you need any help. You will get paid for the work whether your get hired or not.
```
Comment (dhilipsiva) [inline README.md]: @ChillarAnand They will come to our office and work with us for the entire day from our office. This is a paid-trail for a week.

Comment (dhilipsiva) [inline README.md]: From Monday to Friday..

**Pull request merged**